sdcm/tester.py

```diff
@@ -1333,19 +1330,15 @@ def stop_resources(self):
                 node.stop_task_threads(timeout=60)
 
         if self.create_stats:
-            self.update_test_details(errors=db_cluster_errors,
+            test_failing_events = self.get_critical_events()
+            self.update_test_details(errors=test_failing_events,
                                      coredumps=db_cluster_coredumps,
                                      )
 
         if db_cluster_coredumps:
```

Comment:
should we also move the coredump to be included ?, or we can now skip it since coredump are error events already ?

Reply:
we can skip, this piece of code is redundant